### ISSUE TITLE
ruby 3.1 support

### ISSUE THREAD

**Issue by such** [appsignal/appsignal-ruby#790]
aliases are not supported by YAML.load by default.
The line below should be changed to: 

```
      configurations = YAML.load(ERB.new(IO.read(config_file)).result, aliases: true)
```

https://github.com/appsignal/appsignal-ruby/blob/f9d5775217400c59a70d98e9aa96e3dcd06cb1f9/lib/appsignal/config.rb#L373

**Comment by tombruijn** [comment]
Hi @such, thanks for the issue. We're working on this in issue #789 and PR #788. I'm closing this issue.

**Comment by tombruijn** [comment]
@such I've just published the fix in version 3.0.18 of the Ruby gem 🚀
